scripts/operations/migrate.ts

```diff
@@ -4,7 +4,8 @@ import { bash, isLocalNetwork, getContractNames } from './utils'
 export default async (network: string = 'development') => {
   await bash(`npx truffle migrate --network ${network}`)
 
-  if (!isLocalNetwork(network)) {
-    await verify(network, ...getContractNames())
-  }
+  // verification library currently doesn't work for solidity v6
```

Comment:
😢

Reply:
created issue https://github.com/hoprnet/hopr-ethereum/issues/67 to keep track of this